coding-concepts.js

```diff
@@ -37,23 +37,23 @@ const languages = ["JavaScript", "Ruby", "Python", "C++"]
 const index = 1
 // console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: "Javascript"
+// b) Verify and explain: My answer was incorrect because I chose index [1] where it should have been index [0] since it starts from [0].
 
 
 // --------------------4) What will this log?
 
 const weekendDays = ["saturday", "sunday"]
 // console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: "SATURDAY" "SUNDAY"
+// b) Verify and explain: My answer was incorrect because this would need to be a string to console.log the .toUpperCase() and output "SATURDAY, SUNDAY". Since it was an array, this output would not work and we would need to find a different way to use .toUpperCase.
```

Comment:
For sure!  To use .toUpperCase in this instance, some sort of iteration would be needed to access each individual element in the array.